nodejs/eks/nodegroup.ts

```diff
@@ -136,6 +136,13 @@ export interface NodeGroupBaseOptions {
      * Desired Kubernetes master / control plane version. If you do not specify a value, the latest available version is used.
      */
     version?: pulumi.Input<string>;
+
+    /**
+     * The instance role ARN to use for this node group. This instance role ARN
+     * must be supplied along with any other required instance roles to the
+     * cluster's instanceRoleArns'
+     */
+    instanceRoleArn?: pulumi.Input<aws.ARN>;
```

Comment:
Again - would just accept the `Role` directly here.

Reply:
Actually - for even more user-flexibility here - any reason not to accept the `InstanceProfile` instead?  That lets users who care about this provide any profile they want - and avoids the need to construct the profile from the role in this library?